packages/contracts-bedrock/contracts/L1/L1CrossDomainMessenger.sol

```diff
@@ -43,12 +44,17 @@ contract L1CrossDomainMessenger is CrossDomainMessenger, Semver {
      * @inheritdoc CrossDomainMessenger
      */
     function _sendMessage(
+        uint32 _type,
         address _to,
         uint64 _gasLimit,
         uint256 _value,
         bytes memory _data
     ) internal override {
-        PORTAL.depositTransaction{ value: _value }(_to, _value, _gasLimit, false, _data);
+        if (_type == BridgeConstants.BIT_TX){
```

Comment:
I think it is not necessary here to assign different value to `value` according to `deposit type` here.